product/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from .views import ProductView
+
```

Comment:
```suggestion
from django.urls import path

from .views import ProductView

```
django에서 제공하는 클래스와 내가 만든 클래스를 import 할 때 간격 한 줄 넣어주세요 :)

Reply:
넵 수정하겠습니다! 담부턴 꼼꼼하게 확인하겠습니다~